src/twisted/runner/test/test_procmon.py

```diff
@@ -102,6 +102,9 @@ def __init__(self):
 
         self.spawnedProcesses = []
 
+        # set flag to make spawnProcess OSError
+        self._spawn_resource_unavailable = False
```

Comment:
can we have this a public member and document it in a similar way to `@ivar spawnedProcesses` ?